composer.json

```diff
@@ -40,6 +40,7 @@
   "require-dev": {
     "phpunit/phpunit": "^4.8 || ^5.0",
     "fabpot/php-cs-fixer": "^1.11",
+    "satooshi/php-coveralls": "dev-master",
```

Comment:
1.0 is finally released, so we should update to use ^1.0, same goes for all other prooph-components